src/server/service/match.ts

```diff
@@ -89,6 +91,19 @@ export class Match {
 
     this._players = lodash.shuffle([...botPlayers, ...humanPlayers]);
     this._playerPreviousMatches = {};
+    this.addMessage(this.getRandomPrompt());
+  }
+
+  private getRandomPrompt(): ChatMessagePayload {
```

Comment:
Does this replace the `INITIAL_HOST_MESSAGE` const from above? if so maybe remove the const